lib/testers/rule-tester.js

```diff
@@ -345,11 +347,11 @@ class RuleTester {
             });
 
             // Freezes rule-context properties.
-            const originalGet = rules.get;
+            const originalGet = eslint.rules.get;
 
             try {
-                rules.get = function(ruleId) {
-                    const rule = originalGet(ruleId);
+                eslint.rules.get = function(ruleId) {
```

Comment:
I realize this was here before, but mutating a `linter` instance from another module seems messy to me. Now that we're using multiple linter instances, Is there a way to do this without mutating the linter?